src/test/scala/uk/gov/nationalarchives/consignmentexport/ValidatorSpec.scala

```diff
@@ -22,7 +22,8 @@ class ValidatorSpec extends ExportSpec {
       "language".some,
```

Comment:
There doesn't appear to be tests for the new `extractFfidMetadata` function added to the `Validator` class?

Reply:
Ah yeah, good point. I've added some in.